Sources/Logger/Logger.swift

```diff
@@ -10,41 +12,42 @@ public struct Logger: Sendable {
     public init(category: String) {
         self.subsystem = "CrossmintSDK"
         self.osLogger = OSLog(subsystem: subsystem, category: category)
+
+        providers = [
+            OSLoggerProvider(category: category),
+            DataDogLoggerProvider(
+                service: category,
+                clientToken: DataDogConfig.clientToken,
+                environment: DataDogConfig.environment
+            )
+        ]
     }
 
     public func debug(_ message: String) {
-        if isRunningInPlayground() {
-            print("🔍 [\(subsystem)] \(message)")
-        }
-
         guard Logger.level == .debug else { return }
-        os_log(.debug, log: osLogger, "%{public}@", message)
+        for provider in providers {
+            provider.debug(message, attributes: nil)
+        }
     }
 
     public func error(_ message: String) {
-        if isRunningInPlayground() {
-            print("❌ [\(subsystem)] \(message)")
-        }
-
         guard Logger.level != .fault else { return }
-        os_log(.error, log: osLogger, "%{public}@", message)
+        for provider in providers {
+            provider.error(message, attributes: nil)
+        }
     }
 
     public func info(_ message: String) {
-        if isRunningInPlayground() {
-            print("ℹ️ [\(subsystem)] \(message)")
-        }
-
         guard [.debug, .info].contains(Logger.level) else { return }
-        os_log(.info, log: osLogger, "%{public}@", message)
+        for provider in providers {
+            provider.info(message, attributes: nil)
+        }
     }
 
     public func warn(_ message: String) {
-        if isRunningInPlayground() {
-            print("⚠️ [\(subsystem)] \(message)")
-        }
-
         guard [.debug, .info, .default].contains(Logger.level) else { return }
-        os_log(.default, log: osLogger, "%{public}@", message)
+        for provider in providers {
+            provider.warn(message, attributes: nil)
+        }
```

Comment:
I'm thinking about the same here, but maybe we need it to do the filtering.. but then again we could make our own enum instead of using `OSLogType` and have a `level.rawValue >= enabledLogLevel.rawValue` check and also bring this class down from 4 to 1 functions